### PR TITLE
[BLOCKED] Update icons

### PR DESCRIPTION
**Pull request by jordankoschei-okta**
Updated the existing icons to the latest versions, added the new icons, and removed old icons.

### CODE REVIEW

File: packages/odyssey-react-mui/src/iconDictionary/Chat.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+// Code automatically generated by svgr; DO NOT EDIT.
+
+import { forwardRef } from "react";
+import { SvgIcon } from "./SvgIcon";
+import type { SvgIconNoChildrenProps } from "./types";
+
+export type ChatIconProps = SvgIconNoChildrenProps;
+
+export const ChatIcon = forwardRef<SVGSVGElement, ChatIconProps>(
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/iconDictionary/Chat.tsx:21]: What is SVGSVGElement?

File: packages/odyssey-react-mui/src/iconDictionary/index.ts [hunk 2]
```diff
@@ -105,72 +141,132 @@ export * from "./InformationCircleFilled";
...
+import { ResumeIcon } from "./Resume";
+export * from "./Resume";
+
 import { SearchIcon } from "./Search";
 export * from "./Search";
 
-import { SettingsIcon } from "./Settings";
-export * from "./Settings";
+import { ServerIcon } from "./Server";
+export * from "./Server";
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/iconDictionary/index.ts:182]: Did we rename `SettingsIcon` to something else?

**Comment by edburyenegren-okta** [comment]
@KevinGhadyani-Okta I think we'll need to amend this ticket to get this work done: https://oktainc.atlassian.net/browse/OKTA-594521

The `svgo` script in `odyssey-icons` creates optimized SVGs from exports in `figma`. That allows `odyssey-icons` to act as a canonical source for _non-component_ icons SVGs.

The `build:icons` script in `odyssey-react-mui` takes those optimized SVGs and generates our Icon components, which is why they all have `// Code automatically generated by svgr; DO NOT EDIT.`.

**However** something in a prior release broke the second script, so automatic icon generation no longer works. We now get this error:

```
TypeError: odyssey/packages/odyssey-icons/src/alert-circle.svg: Cannot destructure property 'componentName' of 'undefined' as it is undefined.
```

**Comment by edburyenegren-okta** [comment]
Closing this as a dupe of https://github.com/okta/odyssey/pull/1764

**Pull request closed without merging**